goal_src/jak1/pc/debug/default-menu-pc.gc

```diff
@@ -755,6 +755,7 @@
          (menu "PS2 settings"
             ;(flag "PS2 Load speed" #f ,(dm-lambda-boolean-flag (-> *pc-settings* ps2-read-speed?)))
             (flag "PS2 Particles" #f ,(dm-lambda-boolean-flag (-> *pc-settings* ps2-parts?)))
+            (flag "PS2 Actor Vis" #f ,(dm-lambda-boolean-flag (-> *pc-settings* ps2-actor-vis?)))
```

Comment:
Can you keep this where it was before? It was much easier to toggle in the parent menu.

Reply:
Sure thing!